File: app/src/main/java/com/lxl/othello/MainActivity.java
package com.lxl.othello;

import android.os.Bundle;
import android.view.View;
import android.view.View.OnClickListener;


public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.main_new_game).setOnClickListener(new OnClickListener() {
            @Override
            public void onClick(View v) {
                LocalGameActivity.navigateFrom(MainActivity.this);
            }
        });
    }
}

File: app/src/main/java/com/lxl/othello/plus/MainActivity.kt
package com.lxl.othello.plus

import android.os.Bundle
import android.view.View
import android.widget.TextView
import com.lxl.othello.plus.update.InAppUpdate

class MainActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.main_new_game).setOnClickListener { LocalGameActivity.navigateFrom(this@MainActivity) }
        findViewById<TextView>(R.id.main_version).setText(BuildConfig.VERSION_CODE.toString())
    }

    private val mInAppUpdate by lazy {
        InAppUpdate(this)
    }

    override fun onResume() {
        super.onResume()
        mInAppUpdate.onResume()
    }
}